src/js/services/forms.js

```diff
@@ -101,11 +101,12 @@ export default App.extend({
     const channel = this.getChannel();
 
     const prefillFormId = this.form.getPrefillFormId();
+    const flowId = this.action.get('_flow');
 
     return Promise.all([
       Radio.request('entities', 'fetch:forms:definition', this.form.id),
       Radio.request('entities', 'fetch:forms:fields', get(this.action, 'id'), this.patient.id, this.form.id),
-      Radio.request('entities', 'fetch:formResponses:latestSubmission', this.patient.id, prefillFormId),
+      Radio.request('entities', 'fetch:formResponses:latestSubmission', this.patient.id, prefillFormId, flowId),
```

Comment:
`flowId` can only be used if `prefill-flow-response` so it should be passed into this function `fetchLatestFormSubmission(formId)`

Reply:
Ah ya, good call. Updated in last push.